.vib/apache-exporter/goss/apache-exporter.yaml

```diff
@@ -0,0 +1,12 @@
+command:
+  check-apache-exporter-help:
+    exec: apache_exporter --help
+    exit-status: 0
+    stderr:
+      - "usage:"
```

Comment:
Given the version check and the `check-apache-exporter-run` test below, I would say this one doesn't add value to the testing suite, wdyt?